tests/mir-opt/copy-prop/partial_init.rs

```diff
@@ -9,6 +8,9 @@ use core::intrinsics::mir::*;
 // EMIT_MIR partial_init.main.CopyProp.diff
 #[custom_mir(dialect = "runtime", phase = "post-cleanup")]
 pub fn main() {
+    // CHECK-LABEL: fn main(
```

Comment:
I added this FileCheck annotations, but actually this test case doesn't need them since it checks if rustc compiles the function with CopyProp.
I'd like to confirm that we should keep `skip-filecheck`, or go forward with those annotations.

Reply:
The annotations you put are enough.